{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "d18ff20b_186c2621",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 4
      },
      "lineNbr": 0,
      "author": {
        "id": 1071150
      },
      "writtenOn": "2021-03-01T15:35:00Z",
      "side": 1,
      "message": "Thanks Alan.",
      "revId": "46c2aad205d69c19a8ed4f4406bbf89de2002a90",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "073009ae_0a057c91",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 4
      },
      "lineNbr": 0,
      "author": {
        "id": 1071150
      },
      "writtenOn": "2021-03-02T10:15:35Z",
      "side": 1,
      "message": "The TH failures are due to https://b.corp.google.com/issues/181538870.\n\n",
      "revId": "46c2aad205d69c19a8ed4f4406bbf89de2002a90",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}